chat-commands.js

```diff
@@ -2403,10 +2406,14 @@ exports.commands = {
 				this.add(`|unlink|${hidetype}${targetUser.prevNames[prevName]}`);
 			}
 		} else {
-			this.addModAction(`${name}'s messages were cleared from ${room.title} by ${user.name}.`);
+			if (targetUser) {
+				this.addModAction(`${targetUser.name}'s messages were cleared from ${room.title} by ${user.name}.`);
+			} else {
+				this.addModAction(`${target}'s messages were cleared from ${room.title} by ${user.name}.`);
+			}
+			userid = toId(this.inputUsername);
```

Comment:
Why do you set `let userid = targetUser ? targetUser.getLastId() : "";` if you're just going to overwrite it here?